frontend/public/components/modals/modify-application.tsx

```diff
@@ -0,0 +1,90 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { Title } from '@patternfly/react-core';
+import { updateResourceApplication } from '@console/shared/src';
+import FormSection from '@console/dev-console/src/components/import/section/FormSection';
+import ApplicationSelector from '@console/dev-console/src/components/import/app/ApplicationSelector';
```

Comment:
Fully agreed.

Let's create new extension instead of coupling core Console code to specific plugins.

AFAIK we'd also need this in KubeVirt, cc @mareklibra

Reply:
> I think we should add an extension to allow contributing to the action menu of resources.

It's also needed for OCS. +1 to adding an extension point